example/eventsource-polyfill.js

```diff
@@ -7264,6 +7264,19 @@ function hasBom (buf) {
   })
 }
 
+/**
+ * Wrap a callback to ensure it can only be called once.
+ */
+function once(cb) {
+  let called = false
```

Comment:
It sure is nice to be single threaded

Reply:
It is very nice indeed. (at least the illusion of a single thread.)